rclcpp/include/rclcpp/node_options.hpp

```diff
@@ -252,24 +252,25 @@ class NodeOptions
   NodeOptions &
   allow_undeclared_parameters(bool allow_undeclared_parameters);
```

Comment:
Maybe add to documentation that "this option doesn't apply to parameter_overrides", and even elaborate further that "Parameter overrides can never be treated as undeclared parameters".

Reply:
I added a note, see: 870998d69536c339b38d035d55f0648ae841b101